net-misc/r8125/r8125-9.011.01-r1.ebuild

```
@@ -0,0 +1,50 @@
+# Copyright 1999-2023 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+inherit linux-mod-r1
+
+DESCRIPTION="r8125 vendor driver for Realtek RTL8125 PCI-E NICs"
+HOMEPAGE="https://www.realtek.com/en/component/zoo/category/network-interface-controllers-10-100-1000m-gigabit-ethernet-pci-express-software"
+
+SRC_URI="http://rtitwww.realtek.com/rtdrivers/cn/nic1/${P}.tar.bz2"
+
+LICENSE="GPL-2"
```

Happened to loo at a .c file and seen there's a "any later version"
```suggestion
LICENSE="GPL-2+"
```

---

There are conflicting terms in each and every file.
The first line is `SPDX-License-Identifier: GPL-2.0-only` and below in the same file `either version 2 of the License, or (at your option) any later version.`
For example, see:
* https://github.com/awesometic/realtek-r8125-dkms/blob/9.011.00-1/src/r8125_n.c#L1
* https://github.com/awesometic/realtek-r8125-dkms/blob/9.011.00-1/src/r8125_n.c#L12C24-L13C21

As `GPL-2.0+` sources could be used under `GPL-2.0-only` terms, I'd leave `GPL-2.0-only` which fits unambiguously.

---

I see, when there's both I typically just list both rather than combine.

---

The union of `GPL-2.0-only` and `GPL-2.0+` is `GPL-2.0-only`. :)
Listing both terms as both applicable at the same time (not just one of them) is the same as the `-only`.

---

Maybe we should ignore "SPDX identifier" as the full licence wording in the same file has the full force, while "SPDX identified" does not enforce anything legally?